lib/cisco_node_utils/bridge_domain.rb

```diff
@@ -139,62 +139,13 @@ def default_fabric_control
       config_get_default('bridge_domain', 'fabric_control')
     end
 
-    # Bridge-Domain member vni assigning case
-    # Not all the bds created or initialized in this class context can have the
-    # member vni's mapped. So will get the vnis of the ones which are mapped to
-    # the bds.
-    # Eg: Suppose the bd mapping is as below on the switch
-    # bridge-domain 100,105,107-109,150
-    #   member vni 5000, 8000, 5007-5008, 7000, 5050
-    # If puppet layer tries to get values of 100-107 bds the final_bd_vni map
-    # which is returned will contain only these mappings as
-    # {100=>5000,101=>0,102=>0,103>0,104=>0,105=>8000,106=>0,107=>5007}
-    def member_vni
-      final_bd_vni = {}
-      curr_vni = config_get('bridge_domain', 'member_vni')
-      curr_bd_vni = config_get('bridge_domain', 'member_vni_bd')
-      return final_bd_vni if curr_vni.empty? || curr_bd_vni.empty?
-
-      curr_vni_list = BridgeDomain.bd_ids_to_array(curr_vni)
-      curr_bd_vni_list = BridgeDomain.bd_ids_to_array(curr_bd_vni)
-      input_bds = BridgeDomain.bd_ids_to_array(@bd_ids)
-
-      hash_map = Hash[curr_bd_vni_list.zip(curr_vni_list.map)]
-      input_bds.each do |bd|
-        final_bd_vni[bd.to_i] =
-            hash_map.key?(bd.to_i) ? hash_map[bd.to_i] : 0
-      end
-      final_bd_vni
-    end
-
-    # This member_vni mapping will be executed only when the val is not empty
-    # else it will be treated as a 'no' cli and executed as required.
-    # If the mappings do not match in any fashion then cli normally returns a
-    # failure which will be handled.
-    def member_vni=(val)
-      val = val.to_s
-      Feature.vni_enable
-      if val.empty?
-        bd_val = member_vni.keys.join(',')
-        vni_val = member_vni.values.join(',')
-        return '' if vni_val.empty?
-        config_set('bridge_domain', 'member_vni', vnistate: 'no', vni: vni_val,
-                   bd: bd_val, membstate: 'no', membvni: vni_val)
-      else
-        config_set('bridge_domain', 'member_vni', vnistate: '', vni: val.to_s,
-                   bd: @bd_ids, membstate: '', membvni: val.to_s)
-      end
-    end
-
-    def default_member_vni
-      config_get_default('bridge_domain', 'member_vni')
-    end
-
     # Bridge-Domain name assigning case
     # bridge-domain 100
     #   name PepsiCo
     def bd_name
-      config_get('bridge_domain', 'name', bd: @bd_ids)
+      res = config_get('bridge_domain', 'name', bd: @bd_ids)
```

Comment:
Use the same name in the yaml that you use for the getter/setter; e.g.

`config_get('bridge_domain', 'bd_name', bd: @bd_ids)`